src/Basics.lidr

```diff
@@ -226,13 +226,20 @@ members `False` and `True`.
 
 ```idris
 ||| Boolean Data Type
+data Bool = True | False
+```
+
+This definition is written in the simplified style, similar to `Day`. It can 
+also be written in the verbose style (which is more powerful):
```

Comment:
How about removing the bracketed text and adding after the snippet:

> The verbose style is more powerful because it allows us to specify various constraints on constructors. We will see this later on.